### ISSUE TITLE
[low] Color Contrast: Set password: Red text against light grey background fails minimum contrast requirements

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5527]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Navigate to the 'Enter your phone or email' field and enter a valid email address, then click on 'Continue' button.
3. Open the received email and click on 'here' link.
4. Once the 'Set password' page opens, enter a password that does not match the requirements into 'Enter a password' text field.
5. Measure the color contrast ratio of the red 'Your password must have at least 8 characters, 1 capital letter, 1 lowercase letter, 1 number' text against light grey background.

## Expected Result:
The contrast ratio of the text against the background should be at least 4.50:1 for small text and 3.00:1 for large text (14pt bold or 18 pt+).

## Actual Result:
The red 'Your password must have at least 8 characters, 1 capital letter, 1 lowercase letter, 1 number' text (#FC3826) against the light grey background (#FAFAFA) has a color contrast ratio of 3.50:1.

HTML:
`<div dir="auto" class="css-901oao" style="color: rgb(252, 56, 38); font-family: GTAmericaExp-Regular; font-size: 13px; line-height: 18px; margin-top: 4px; margin-bottom: 4px;">Your password must have at least 8 characters,
1 capital letter, 1 lowercase letter, 1 number.</div>`

CSS:
`element.style {
color: rgb(252, 56, 38);
font-family: GTAmericaExp-Regular;
font-size: 13px;
line-height: 18px;
margin-top: 4px;
margin-bottom: 4px;
}`

## Workaround:
No workaround

## Area issue was found in:
Set password page

## Failed WCAG checkpoints
1.4.3

## User impact:
The low-vision and color-blind users might not be able to see the text.

## Suggested resolution:
Use a darker shade of red for text to achieve a color contrast ratio of at least 4.50:1. One example of color to use is `#E21603`, which will give a color contrast ratio of 4.62:1.

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5183580_color_contrast_-_red_text](https://user-images.githubusercontent.com/88733897/134780300-271b808d-10fb-4fd8-bcae-a2564b16fadd.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
